lib/va1010_forms/utils.rb

```diff
@@ -38,5 +37,24 @@ def soap
     def override_parsed_form(parsed_form)
       HCA::OverridesParser.new(parsed_form).override
     end
+
+    private
+
+    def submission_body(formatted_form)
+      content = Gyoku.xml(formatted_form)
+      submission_body = soap.build_request(:save_submit_form, message: content).body
+      log_payload_info(formatted_form, submission_body)
+
+      submission_body
+    end
+
+    def log_payload_info(formatted_form, submission_body)
+      form_name = formatted_form['va:form']['va:formIdentifier']['va:value']
```

Comment:
Yeah that's probably a good idea. I did test that for attachments below and included the safe navigator for it, but it can't hurt to add it for this one as well. I'm not familiar enough with the full structure of the form to know what is always there vs optional.

Reply:
Might want to use [Hash#dig](https://ruby-doc.org/core-3.1.1/Hash.html#method-i-dig) to make the key chain safer, too?
```suggestion
      form_name = formatted_form.dig('va:form', 'va:formIdentifier', 'va:value')
```
This way if e.g. `formatted_form['va:form']['va:formIdentifier']` is `nil` then it returns `nil` instead of failing with `undefined method \`[]' for nil:NilClass`

Reply:
Appreciate the suggestions! I updated it to use dig here. It appears this scenario is highly unlikely, since we set the `va:value` in the enrollment_system.rb in a ternary, but I am all for ensuring a more robust solution.